### DISCUSSION TITLE
buddhism/delusion/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#20] [General]
# buddhism/delusion/

https://shanenull.com/buddhism/delusion/

**Comment by shane0** [comment]
https://www.rigpawiki.org/index.php?title=Two_kinds_of_ignorance

**Comment by shane0** [comment]
https://science-atlas.com/faq/what-law-of-thermodynamics-is-entropy/

**Comment by shane0** [comment]
**Anatta in the style of Cormac McCarthy:**

The self is an illusion.

It is a mirage in the desert of samsara.

It is a fleeting shadow in the moonlight.

Anatta is the Buddhist teaching that there is no permanent or unchanging self.

There is only a collection of ever-changing phenomena.

The body, the mind, and the emotions are all impermanent.

They are all constantly arising and passing away.

There is no central core, no essential self.

Only the flow of phenomena.

To see the truth of anatta is to see the world as it really is.

It is to see the world without the filter of the ego.

It is to see the world with compassion and understanding.

Anatta is not a nihilistic teaching.

It is not a teaching of despair.

It is a teaching of liberation.

When we see the truth of anatta, we are free from the suffering caused by attachment to the self.

We are free from the need to prove ourselves, to be better than others, and to accumulate possessions.

We are free to simply be.

So let us meditate on anatta.

Let us contemplate the impermanence of all things.

And let us find liberation in the realization that there is no permanent self.

**Note:** I have taken some liberties with the traditional Buddhist explanation of anatta in order to fit the style of Cormac McCarthy.

**Comment by shane0** [comment]
https://www.dhammatalks.org/suttas/KN/StNp/StNp4_2.html
